app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RecipeIngredientsController < ApplicationController
+  
+    def create
+      @recipe = Recipe.find([params[:id]])
+      if params[:ingredient_id] != nil
```

Comment:
Great idea with the nil check here. If you didn't have a nil check during an assessment, that's quite alright (we don't much sad path handling) but I'm curious why you have a nil check for ingredients and not recipes? Maybe this was related to your comment around starting this joins record in a different controller.